util/uploadTestResults.js

```diff
@@ -25,22 +25,44 @@ const uploadTestResults = (env, results, done) => {
       "data": results.slice(i, i + CHUNK_SIZE),
     }
 
-    debug(`Posting to Test Analytics: ${JSON.stringify(data)}`)
+    if (Debug.enabled()){
+      axios.interceptors.request.use(function (config) {
+        Debug.log(`Test Analytics Sending: ${JSON.stringify(config)}`);
+        return config;
+      }, function (error) {
+        if (error.response) {
+          Debug.log(`Test Analytics request error: ${error.response.status} ${error.response.statusText} ${JSON.stringify(error.response.data)}`);
+        } else {
+          Debug.log(`Test Analytics request error: ${error.message}`)
+        }
```

Comment:
Is the response handled by the response interceptor below? I see this block is the same as the error block in the response interceptor below?

If we intend to keep both, is it worth differentiating the debug logs a little so we know which line reported the log?

Reply:
I'm not sure if the `error` object will ever have a response key in this situation, but I decided to try and log it since this is inherently a big unknown area and more logging is probably a good idea.

I included "request error" in the error strings to help differentiate them. Is that what you're after @gchan?

Reply:
That is perfect. Thanks!